pkg/recibo/logger_test.go

```diff
@@ -0,0 +1,55 @@
+package recibo
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestLogger(t *testing.T) {
+	testLogFile := "/tmp/test.log"
+
+	logPathEnvValue, logPathEnvExists := os.LookupEnv("LOGFILEPATH")
+	logLevelEnvValue, logLevelEnvExists := os.LookupEnv("LOGLEVEL")
+	os.Setenv("LOGFILEPATH", testLogFile)
+	os.Setenv("LOGLEVEL", "Debug")
+
+	logger := NewLogger()
+
+	logger.Debug("Prueba DEBUG")
+	logger.Info("Prueba INFO")
+	logger.Warn("Prueba WARN")
+	logger.Error("Prueba ERROR")
+
+	if logPathEnvExists {
+		os.Setenv("LOGFILEPATH", logPathEnvValue)
+	} else {
+		os.Unsetenv("LOGFILEPATH")
+	}
+	if logLevelEnvExists {
+		os.Setenv("LOGLEVEL", logLevelEnvValue)
+	} else {
+		os.Unsetenv("LOGLEVEL")
+	}
```

Comment:
Si usaras Go 1.17, podrías establecer las variables de entorno en tests usando [esta](https://khanakia.medium.com/go-1-17-new-features-added-to-testing-9a8862cb03c0) nueva funcionalidad, que puede evitar problemas de contaminación de variables de entorno de un test a otro. Aunque no puedas aplicarlo actualmente tenlo en cuenta para el futuro.

Reply:
No conocía esa funcionalidad y efectivamente parece muy práctica para estos casos. Como ya he comentado un poco más arriba, el proyecto soporta las versiones 1.16 y 1.17 y creo que probablemente no sea conveniente dejar de dar soporte a la 1.16 solo para incluir esto, pero me lo anoto para el futuro. Gracias por la aportación!